### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by rkapoor10**
- folder structure 
folder-structure

- wishlist page
ss

### CODE REVIEW

File: src/components/cart/CartProducts.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    return(
+        <>
+            <div className="child cart-pg-product">
+            <div className="space-between alignitems-c wrapit seperator-lightgray padtb-1 mb-2">
+                <h4 className="baseteal">MY CART (3)</h4>
+                <span className="darkteal txt-s alignitems-c"><MdLocationPin/> Deliver To</span>
+            </div>
+
+
+
```
Comment (mariyasada) [inline src/components/cart/CartProducts.jsx:15]: remove this space if not neccessary

File: src/pages/cart/CartPage.jsx [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
-        <h1>This is Cart Page</h1>
-    )
-}
+const CartPage = () => {
+  return (
+    <>
+      <div className="cart-pg-layout">
+        <div className="child cart-pg-navbar">
+          <Navbar />
+        </div>
```
Comment (mariyasada) [inline src/pages/cart/CartPage.jsx:13]: why have you placed the navbar here?

**Review by mariyasada — APPROVED** [review]
good to merge

**Review by tusharkandpal — APPROVED** [review]
Put the `Navbar` & `Footer` components out of routes since they are common for all screens.

Code structure is excellent!

**Review by ajit-sharmaGH — APPROVED** [review]

File: src/components/cart/CartProducts.jsx (hunk 1 above)
Comment (rkapoor10) [reply src/components/cart/CartProducts.jsx:15]: ok

File: src/pages/cart/CartPage.jsx (hunk 2 above)
Comment (rkapoor10) [reply src/pages/cart/CartPage.jsx:13]: as they have separate grid styles...will try to refactor and have only once

**Pull request merged**